scripts/generate_zoo_yml.py

```diff
@@ -0,0 +1,57 @@
+import structlog
+
+from zoo.repos.github import create_remote_commit as create_github_remote_commit
+from zoo.repos.github import get_file_content as get_github_file_content
+from zoo.repos.gitlab import create_remote_commit as create_gitlab_remote_commit
+from zoo.repos.gitlab import get_file_content as get_gitlab_file_content
+from zoo.repos.models import Repository
+from zoo.repos.zoo_yml import generate, parse, validate
+from zoo.services.models import Service
+
+log = structlog.get_logger()
+
+
+def file_exists(remote_id, path, provider, ref="master"):
+    get_file_content = None
+    if provider == "github":
+        get_file_content = get_github_file_content
+    else:
+        get_file_content = get_gitlab_file_content
+
+    try:
+        _ = get_file_content(remote_id, path, ref)
+        return True
+    except Exception:
+        return False
+
+
+def generate_file(remote_id, message, actions, branch, provider):
+    create_remote_commit = None
```

Comment:
And actually seeing that here it's done in the same way it might make more sense to get the remote only once and pass it as a variable, or just avoid this functions at all, as they aren't doing much more than dispatching based on the provider.

Something like:
```suggestion
def main():
    zooyml = ".zoo.yml"
    remote_id, provider = service.repository.remote_id, remotes[service.repository.provider]
    try:
        remote.get_file_content(remote_id, zooyml, "master")
        continue
    except FileNotFoundError:
        log.debug("scripts.generate_zoo_yml", ".zoo.yml file not found, proceeding with creation")
        
    yml = generate(service)
    
    actions = [{"action": "create", "content": yml, "file_path": zooyml}]
    message = "feat(zoo): add .zoo.yml file"
    branch = "master"
    
    provider.create_remote_commit(remote_id, message, actions, branch)
```

Reply:
Looks good to me.

Reply:
This is implemented in the django command.